linux

```diff
@@ -26,19 +26,28 @@ gradle:6.4 gradle-6.4 DEPRECATED
 gradle:6.4-jdk11 gradle-6.4-jdk11 DEPRECATED
 gradle:6.4-jdk14 gradle-6.4-jdk14 DEPRECATED
 gradle:6.4-jdk8 gradle-6.4-jdk8 DEPRECATED
-gradle:jdk8 gradle-jdk8
-gradle:jdk11 gradle-jdk11
+gradle:jdk8 gradle-jdk8 DEPRECATED
+gradle:jdk11 gradle-jdk11 DEPRECATED
 gradle:jdk12 gradle-jdk12 DEPRECATED
 gradle:jdk13 gradle-jdk13 DEPRECATED
 gradle:jdk14 gradle-jdk14 DEPRECATED
 gradle:jdk16 gradle-jdk16 DEPRECATED
-gradle:jdk17 gradle-jdk17
+gradle:jdk17 gradle-jdk17 DEPRECATED
 gradle:jdk18 gradle-jdk18 DEPRECATED
 gradle:jdk19 gradle-jdk19 DEPRECATED
 gradle:jdk20 gradle-jdk20 DEPRECATED
-gradle:jdk21 gradle-jdk21
-gradle:jdk23 gradle-jdk23
-gradle:jdk24 gradle-jdk24
+gradle:jdk21 gradle-jdk21 DEPRECATED
+gradle:jdk23 gradle-jdk23 DEPRECATED
+gradle:jdk24 gradle-jdk24 DEPRECATED
+gradle:8-jdk8 gradle-8-jdk8
+gradle:8-jdk11 gradle-8-jdk11
+gradle:8-jdk17 gradle-8-jdk17
+gradle:8-jdk21 gradle-8-jdk21
+gradle:8-jdk24 gradle-8-jdk24
+gradle:9-jdk17 gradle-9-jdk17
+gradle:9-jdk21 gradle-9-jdk21
+gradle:9-jdk23 gradle-9-jdk23
+gradle:9-jdk24 gradle-9-jdk24
```

Comment:
@calhar-snyk FYI, I preferred explicit over implicit, also pinning the version 9s. OK by you?